src/connector/src/sink/iceberg/java/src/main/java/com/risingwave/iceberg/SparkIcebergCompaction.java

```diff
@@ -0,0 +1,26 @@
+package com.risingwave.iceberg;
```

Comment:
Should we also include the patch to iceberg code in this PR?

Reply:
I think we should maintain our own iceberg repo and the patch should be applied to that repo.

Reply:
I put the patch here. https://github.com/risingwavelabs/iceberg